### PR TITLE
Remove sticky class buttons completely

### PR DESCRIPTION
**Pull request by tfennelly**
# Description

See [JENKINS-37918](https://issues.jenkins-ci.org/browse/JENKINS-37918).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [x] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@jenkinsci/code-reviewers @reviewbybees

### CODE REVIEW

**Comment by tfennelly** [comment]
https://github.com/jenkinsci/blueocean-plugin/tree/JENKINS-35847

**Comment by tfennelly** [comment]
See https://github.com/jenkinsci/blueocean-acceptance-test/tree/gh-pages

File: src/main/js/custom_commands/moveClassicBottomStickyButtons.js [hunk 1]
```diff
@@ -25,47 +25,72 @@ Cmd.prototype.command = function () {
                     // In classic Jenkins, we can try dipping into the js-modules
                     // and get jQuery. If it's not there, then we're not in classic Jenkins
                     // and we don't care.
-                    var $ = window.jenkinsCIGlobal.plugins["jquery-detached"].jquery2.exports;
-                    doTweaks($);
+                    var $jquery = window.jenkinsCIGlobal.plugins["jquery-detached"].jquery2.exports;
```
Comment (scherler) [inline src/main/js/custom_commands/moveClassicBottomStickyButtons.js]: REALLY, weapons of mass destructions

Comment (tfennelly) [inline src/main/js/custom_commands/moveClassicBottomStickyButtons.js]: @scherler What? And in caps and all 😄

**Pull request merged**